Formula/slither-analyzer.rb

```diff
@@ -6,6 +6,7 @@ class SlitherAnalyzer < Formula
   url "https://files.pythonhosted.org/packages/d9/18/4af93281e1e7ae22fb2bdc3a3da36397781b8b99611fa54379f5a6d1631d/slither-analyzer-0.9.6.tar.gz"
   sha256 "3ee724a3bbd29d631d9c4ed766a63bcd18ed39775b39796b19f02edf4de3fe80"
   license "AGPL-3.0-only"
+  revision 1
```

Comment:
Do users need this?

Reply:
the dependency got changed though, so I feel it it would be better to bump revision, I am tempting on this (also ok with removing it)

Reply:
i guess based on the release notes of cytoolz, i think we can remove revision. I will do that.